bluepysnap/edges.py

```diff
@@ -506,7 +456,7 @@ def property_dtypes(self):
         Returns:
             pandas.Series: series indexed by field name with the corresponding dtype as value.
         """
-        return self.properties([0], list(self.property_names)).dtypes.sort_index()
+        return self.get([0], list(self.property_names)).dtypes.sort_index()
 
     def container_property_names(self, container):
         """Lists the ConstContainer properties shared with the EdgePopulation.
```

Comment:
At the line 475 there is a call to `my_edge_population.property_values`. Is there such method of EdgePopulation?

Reply:
no !